src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -88,6 +88,94 @@ def get(self, request, organization):
         return Response(baseline_data)
 
 
+class OrganizationEventsTrace(OrganizationEventsEndpointBase):
+    def get(self, request, organization, trace_id):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response(status=404)
+
+        # Get 100 events from snuba, and pray these cover the trace
```

Comment:
Is paginating the results here something we should consider for traces with more than 100 transactions?

Reply:
I think this is something to consider with the final product, since we could be paging for a while with a 4k transaction trace. I chose 100 since it should cover the majority of traces (and it coincidentally aligns with the max event_ids for `eventstore.get_events`)